nemo/core/classes/modelPT.py

```diff
@@ -190,14 +190,29 @@ def restore_from(cls, restore_path: str):
                 model_weights = path.join(tmpdir, _MODEL_WEIGHTS)
                 conf = OmegaConf.load(config_yaml)
                 OmegaConf.set_struct(conf, True)
+                cls.update_config_with_specific_artifacts(config=conf, artifacts_dir=tmpdir)
                 instance = cls.from_config_dict(config=conf)
                 instance.load_state_dict(torch.load(model_weights))
 
+                logging.info(f'Model {cls.__name__} was successfully restored from {restore_path}.')
             finally:
                 os.chdir(cwd)
 
         return instance
 
+    @classmethod
+    def update_config_with_specific_artifacts(cls, config: OmegaConf, artifacts_dir: str = None) -> OmegaConf:
```

Comment:
No need for artigscts_dir to be None by default, it is assured provided as tmpdir if it gets called

Reply:
fixed